src/components/TeamPics/teampics.scss

```
@@ -0,0 +1,74 @@
+h1, h2, h3, h4, p {
+    color: black;
+}
+
+h1 {
+    font-weight: bold;
+}
+
+.Header {
+    font-size: 40px;
+    padding: 0% 0% 2% 0%;
+}
+
+/* Most of .Container and .Card code taken from Codepen and edited: https://codepen.io/izer/pen/RYvevG */
+.Component {
+    padding: 7%;
+
+    .Container1 {
+
+        //margin: auto;
+        //margin-top: 50px;
+        //max-width: 1000px;
+        display: grid;
```

```suggestion
        display: flexbox;
```

Flexbox allows easier responsive design than grid, because with grid we would need to change the grid layout for each device screen width.

---

Changed.